repo/packages/A/arangodb/1/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/arangodb/arangodb-dcos"
```

Comment:
we have previously allowed this.. however internally we discussed this and we would prefer to see the link for sub command be for a specific version and not off master.   we don't want changes to the master to be automatically added in without a versioning of the dcos service.   the idea is to have an immutable version.